README.md

```diff
@@ -48,6 +48,7 @@ latitudeShApi.Profile.get().then((response) => {
 - `Plans.get`. Params: `(planId, searchParams)`. [Reference](https://docs.latitude.sh/reference/get-plan)
 - `Plans.list`. Params: `(searchParams)`. [Reference](https://docs.latitude.sh/reference/get-plans)
 - `Plans.operatingSystems`. Params: `(searchParams)`. [Reference](https://docs.latitude.sh/reference/get-plans-operating-system)
+- `Plans.storages`. Params: `(searchParams)`. [Reference](https://docs.latitude.sh/reference/get-storage-plans)
```

Comment:
is the plural naming l here necessary? why not `Plans.storage` ?

Reply:
Just did the plural for standardization since `operatingSystems` was on plural.

Reply:
I agree it should be `Plans.storage`

Reply:
solved! ty guys!